compiler/test/input/mixedPatternMatching.gr

```diff
@@ -30,14 +33,14 @@ let test = test => {
       [PList([a]), PAssoc({ foo: [b], _ }), PMulti([], c, { foo: [d], _ })]
     ) =>
       a +
-      b +
-      c +
-      d,
+        b +
+        c +
+        d,
```

Comment:
I don't think it's the worst... we deliberately made it this way some time ago, largely because of statements:
```
assert reallyLongFoo &&
  reallyLongBar
```

Could probably be better though/maybe only on statements.

Reply:
But why are these even wrapping in the first place?

Reply:
I believe it's because they are not grouped, and anything not grouped will wrap if anything on the line wraps (I think).

(I didn't change anything to do with this by the way, I just formatted the file.)

Reply:
That's fine that you didn't change anything. I just think that code review is a good time to point out things that aren't great about the state of things. Can we get an issue? 🙏

Reply:
https://github.com/grain-lang/grain/issues/1634